.github/workflows/support.yml

```diff
@@ -41,6 +41,32 @@ jobs:
           GENERATED_FILE=docs/HANDLED_RULES.md
           python -m sorald.${SCRIPT_NAME} -o ${GENERATED_FILE}
 
+          ./.github/submit-pr.sh \
+            --branch-prefix   ${SCRIPT_NAME} \
+            --generated-file  ${GENERATED_FILE} \
+            --gh-sha          ${{ github.sha }} \
+            --gh-token        ${{ secrets.GITHUB_TOKEN }} \
+            --gh-repository   ${{ github.repository }} \
+            --gh-workflow     ${{ github.workflow }}
+      - name: Generate usage/repair.md and submit a PR if modified
+        run: |
+          SCRIPT_NAME=usage
+          GENERATED_FILE=docs/usage/repair.md
+          python -m sorald.${SCRIPT_NAME} -s repair -o ${GENERATED_FILE}
+          
+          ./.github/submit-pr.sh \
+            --branch-prefix   ${SCRIPT_NAME} \
+            --generated-file  ${GENERATED_FILE} \
+            --gh-sha          ${{ github.sha }} \
+            --gh-token        ${{ secrets.GITHUB_TOKEN }} \
+            --gh-repository   ${{ github.repository }} \
+            --gh-workflow     ${{ github.workflow }}
+      - name: Generate usage/mine.md and submit a PR if modified
+        run: |
+          SCRIPT_NAME=usage
+          GENERATED_FILE=docs/usage/mine.md
+          python -m sorald.${SCRIPT_NAME} -s mine -o ${GENERATED_FILE}
+          
```

Comment:
@monperrus @khaes-kth we talk about diffs of code are not clear. Look at this. I added two separate jobs, but it fused the first job with one previously added.

Reply:
This diff is not in the scope of diffmin of course, but I am just showing examples of how diffs can be unclear at times.

Reply:
indeed, a motivating example to be added somewhere